bin/create.js

```diff
@@ -219,84 +227,70 @@ function handleCreate (argv) {
                 argv.name = md5.update(fol, 'utf8').digest('hex');
             }
         }
-    }
-    else if (argv.name && argv.output !== 'none') {
-        throw new Error('The `name` option can only be specified when --output is set to `url`.');
+    } else if (argv.name && argv.output !== 'none') {
+        // throw Cli.usageError('The `name` option can only be specified when --output is set to `url`.');
```

Comment:
Yes, this is commented out on purpose as it prevented the cron webtask (named) from being created in the latest changes in ops.

However, with your suggestion to indicate the actual commands for each core webtask, we can probably uncomment this as the command construction would not be shared across all core webtasks (both named and not named).